### PR TITLE
added group names in the Readme.md file

### PR DESCRIPTION
**Pull request by conwaycolin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 Practice space for using the Github Workflow.
 
 Participants:
+Colin Conway
```
Comment (smithjac) [inline README.md]: Don't really like the name here, change it to something better if possible

Comment (conwaycolin) [reply README.md]: whats your middle name? Trying to add those

Comment (smithjac) [reply README.md]: Looks fantastic, great work

File: README.md [hunk 2]
```diff
@@ -4,7 +4,9 @@
 Practice space for using the Github Workflow.
 
 Participants:
-Ben Coleman
+Colin Carl Conway
```
Comment (smithjac) [inline README.md:7]: Great work, fantastic job

**Pull request merged**